src/components/ClockFace.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import TimerPanel from './TimerPanel';
+
+const ClockFace = () => {
+  // currentSession is the index of the allSessions array
+  const [currentSession, setCurrentSession] = useState(0);
+  // AllSession's structure: [[min, sec], [min, sec], [min, sec], [min, sec]]
+  // these will be set by the user
+  const [allSessions, setAllSessions] = useState([[0, 10], [1, 0], [0, 20], [0, 30]]);
+
+  const handleNextSession = () => {
+    if (allSessions.length - 1 === currentSession) {
+      // this is the last session, so go back to display the first session, without auto starting
```

Comment:
Parent <ClockFace /> doesn't determines if a timer auto-starts or not, its decided in the child.